Terminal.Gui/Views/StatusBar.cs

```diff
@@ -105,8 +105,8 @@ private Action<Application.ResizedEventArgs> Application_Resized ()
 			return delegate {
 				X = 0;
 				Height = 1;
-				if (SuperView == null || SuperView is Toplevel) {
-					Y = SuperView.Frame.Height - 1;
+				if (SuperView != null || SuperView is Toplevel) {
```

Comment:
@BDisp I don't think what I did here is right. It fixes a bug where this crashes if `StatusBar` was not added to toplevel, but the logic seems off. 

Can you help?

Reply:
I tested with both and had no crashes. Where this is more controlled is in the EnsureVisibleBounds method. Maybe the crashes was due with this method.